package.json

```diff
@@ -81,7 +81,7 @@
         "@swc/core": "^1.3.11",
         "@swc/jest": "^0.2.23",
         "@telefonica/acceptance-testing": "2.13.0",
-        "@telefonica/eslint-config": "^1.0.7",
+        "@telefonica/eslint-config": "^1.3.0",
```

Comment:
unrelated with this task, but I've updated the eslint-config and fixed some type imports